packages/flutter/test/gestures/drag_test.dart

```diff
@@ -1676,22 +2064,19 @@ testGesture('Touch drags should allow pointer pan/zooms to join them', (GestureT
     expect(updatedScrollDelta, isNull);
     expect(didEndPan, isFalse);
 
-    // Gesture will be claimed when distance reaches kPanSlop, which was 36.0 when this test was last updated.
-    tester.route(panZoomPointer.panZoomUpdate(const Offset(10.0, 10.0), pan: const Offset(20.0, 20.0))); // moved 20 horizontally and 20 vertically which is 28 total
-    expect(didStartPan, isFalse);
-    expect(updatedScrollDelta, const Offset(20.0, 20.0));
-    updatedScrollDelta = null;
-    expect(didEndPan, isFalse);
+    // Here are two pointers, so when they both moved, the gesture will move finally.
+    // one pointer moves (10, 10), and another moves (30, 30), the result is (30, 30).
+    tester.route(touchPointer.move(const Offset(80.0, 80.0)));
```

Comment:
Why is this line moving by (80, 80) instead of (30, 30) as in the comment?

Reply:
The Start location is (70, 70) and the End location is (80, 80), so the `touchPointer` moves by (10, 10). The (30, 30) is from the `panZoomPointer`.